test/test_db.py

```diff
@@ -35,6 +34,7 @@
 def db():
     config = MockConfig()
     config.core.db_filename = db_filename
+    config.core.db_type = 'sqlite'
```

Comment:
I need to update #1781 so it will uses `sqlite` in its new config.

Reply:
`sqlite` is already the default: https://github.com/sopel-irc/sopel/blob/185686f92d299f95027f4dccf40ffd994e7807fd/sopel/config/core_section.py#L133-L135 Did you need to update something else besides the fake config?

Reply:
Nothing else. If it's already the default, then I've nothing to update!